image_definitions/data/Dockerfile

```diff
@@ -16,8 +16,11 @@ RUN echo "https://dl-cdn.alpinelinux.org/alpine/v3.17/community" >> /etc/apk/rep
 RUN apk update
 
 # Add latest version of python  and poetry
-RUN apk add --update python3 py3-pip gcc musl-dev linux-headers python3-dev
+RUN apk add --update python3 py3-pip gcc musl-dev linux-headers python3-dev jq
 RUN pip install pytest pylint pylint-exit pytest-azurepipelines pytest-cov poetry
 
 # Install the databricks CLI, precommit and tomli
-RUN pip install databricks-cli pre-commit tomli
\ No newline at end of file
+RUN pip install databricks-cli pre-commit tomli
+
+# Install Azure CLI in the Data image
+RUN pip install azure-cli
```

Comment:
Please ensure your editor is set to add a trailing newline to files, if you see the no entry ⛔ sign on Github it means the file is missing one.

Trailing newlines are a requirement for some Unix tools else they report less lines

Reply:
Updated now, thanks .